portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -2,19 +2,19 @@
 
 /** an Instance of a comment **/
 public final class Comment {
-  final private String user;
+  final private String username;
```

Comment:
prefer the order "private final" instead of "final private"

Reply:
yes the normal css style is two spaces

Reply:
Thank you! it has been resolved.